raiden_contracts/data

```diff
@@ -0,0 +1 @@
+data_0.8._
```

Comment:
Not required for this PR but eventually you should put all these `data_X.Y.Z` directories under a common `data/` directory.

Reply:
Well, I'm being nice here.  Put `0.8._` in `raiden` codebase and that will keep working for a while.